sonar-project.properties

```diff
@@ -6,7 +6,8 @@ sonar.organization=matrix-org
 
 sonar.sources=src,res
 sonar.tests=test,cypress
-sonar.exclusions=__mocks__,docs
+# instrumentation is disabled on SessionLock
+sonar.exclusions=__mocks__,docs,src/utils/SessionLock.ts
```

Comment:
This should be in `sonar.coverage.exclusions` below - we do not want to disable other analyses on it.

Reply:
fml